src/WebJobs.Script/Diagnostics/DiagnosticEventLoggerExtensions.cs

```diff
@@ -20,7 +20,7 @@ public static void LogDiagnosticEvent(this ILogger logger, LogLevel level, int e
                 { ScriptConstants.ErrorCodeKey, errorCode }
             };
 
-            logger.Log(level, eventId, stateDict, exception, (state, ex) => message);
```

Comment:
`eventId` isn't used anymore... can we remove it from the method completely and just hardcode it as 0?